spouse/spouse_demo.py

```diff
@@ -0,0 +1,363 @@
+# %% [markdown]
+# # Detecting spouse mentions in sentences
+
+# %% [markdown]
+# We will walk through an example text classification task to explore how Snorkel can be used for Information Extraction.
```

Comment:
can we highlight up front the deltas from this to the intro spam tutorial? (we are pulling from even more labeling sources, this is a different task, etc.)